generators/chipyard/src/main/scala/upf/firrtl/Annotations.scala

```diff
@@ -0,0 +1,14 @@
+// See LICENSE for license details
+package chipyard.upf.firrtl
+
+import firrtl.annotations._
+import firrtl.stage.{RunFirrtlTransformAnnotation}
+
+trait UPFAnnotation extends Annotation {
+  val name: String
+}
+
+case class NameUPFAnnotation(target: Target, name: String) extends SingleTargetAnnotation[Target] with UPFAnnotation {
```

Comment:
Since I think this Annotation is a noop, I think you can just make this a `NoTargetAnnotation`: https://github.com/chipsalliance/firrtl/blob/94d425f0f48e84bbae1be9d44d64615a37d960d8/src/main/scala/firrtl/annotations/Annotation.scala#L62